### PR TITLE
Updated Code with Exception Handling

### PR DESCRIPTION
**Pull request by Srishti-Chhabra**
This is the updated code with all the issues fixed, as said. It also includes three methods for handling exceptions.

### CODE REVIEW

File: src/main/scala/srishti/TGTraining.scala [hunk 1]
```diff
@@ -1,6 +1,49 @@
...
+  }
+
+  def divideIfElse(firstNum: Int, secondNum: Int): Option[Int] = {
+    // Handling Exception Using if-else block
+    if(secondNum==0) None
+    else Some(firstNum/secondNum)
+  }
 }
+
+object Main
```
Comment (tg-trainer) [inline src/main/scala/srishti/TGTraining.scala:40]: remove it

Comment (tg-trainer) [reply src/main/scala/srishti/TGTraining.scala:40]: as we have the unit test

**Review by tg-trainer — CHANGES_REQUESTED** [review]
please do the requested changes

File: src/main/scala/srishti/TGTraining.scala (hunk 1 above)
Comment (Srishti-Chhabra) [reply src/main/scala/srishti/TGTraining.scala]: Done Sir!

**Comment by Srishti-Chhabra** [comment]
Updated Code with Exception Handling by Srishti-Chhabra · Pull Request #6 ·
tg-trainer/TGT_Scala_23-24 (github.com)
<https://github.com/tg-trainer/TGT_Scala_23-24/pull/6#discussion_r1386176474>

Done the suggested changes, Sir. Please review them.
Thank you!

**Pull request merged**